docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,247 @@
+the co-polarized returns (also vertically polarized) and the cross-polarized returns (horizontally polarized) are 
+measured. This results in both VV and VH RTC-S1 products. 
+
+In some areas, particularly remote islands and regions close to the Arctic Ocean, acquisitions are collected with a 
+horizontal primary polarization. This results in HH and HV RTC-S1 products. 
+
+Different polarizations are sensitive to different surface characteristics, with VV being more sensitive to surface 
+roughness, VH or HV being more sensitive to volume scatterers such as vegetation, and HH being more sensitive to 
+double-bounce scattering from objects that stand perpendicular to the ground. 
+
+### Archived OPERA RTC-S1 Products
+
+The OPERA project has generated RTC-S1 products for all Sentinel-1 acquisitions over landmasses (except Antarctica) 
+since January 1, 2022. RTC-S1 products are output for all available polarizations. As new Sentinel-1 SLCs are 
+acquired and added to ASF's archive, the OPERA project continues to process them to RTC-S1 products.  
+
+RTC-S1 products generated by the OPERA project are all archived by ASF and can be accessed using a number of methods: 
+
+  - ASF's [Vertex Data Search](https://search.asf.alaska.edu/#/?dataset=OPERA-S1&productTypes=RTC "search.asf.alaska.edu" ){target=_blank}, 
+    which is a user-friendly and map-centric interface specialized for search and discover of ASF's SAR holdings
+  - Programmatically using the [asf_search python package](https://docs.asf.alaska.edu/asf_search/basics/ "docs.asf.alaska.edu" ){target=_blank}
+  - NASA's [Earthdata Search](https://search.earthdata.nasa.gov/search/granules?p=C2777436413-ASF "search.earthdata.nasa.gov" ){target=_blank} 
+    interface, which provides access to all of NASA's Earth Science datasets
+
+For more information on options for accessing archived OPERA RTC-S1 products, refer to ASF's 
+[OPERA Sentinel-1 RTC Tutorial](https://storymaps.arcgis.com/stories/dc2807b444924fc3a76c117a2c909f8b "OPERA Sentinel-1 RTC StoryMap Tutorial" ){target=_blank}. 
+For more information on the technical specifications of the OPERA RTC-S1 products, refer to JPL's 
+[RTC Product Documentation](https://www.jpl.nasa.gov/go/opera/products/rtc-product/ "OPERA RTC Product" ){target=_blank}.
+
+### OPERA RTC-S1 Products On Demand
+
+You can also order OPERA RTC-S1 products from ASF on demand. This is particularly useful if you need a time 
+series of RTC-S1 products that extends beyond the start of the archive. On-Demand processing is available for any 
+Sentinel-1 burst with the same burst ID (footprint) as an existing OPERA RTC-S1 product. 
+
+On-Demand RTC-S1 products are generated using the same code that is used by the OPERA project, but are processed 
+using ASF's 
+[HyP3](https://hyp3-docs.asf.alaska.edu/){target=_blank} 
+cloud-native processing platform instead of JPL's 
+[OPERA SDS](https://software.nasa.gov/software/NPO-52101-1){target=_blank} 
```

Comment:
Should "SDS" be spelled out? (Science Data System)? I can see it also being fine as-is since it's their product.

Reply:
I thought I'd spelled it out prior, but I may have removed the section that referenced it. Thanks for the catch!